bubble-wrap.yaml

```diff
@@ -2713,8 +2713,17 @@ layers:
                 lines:
                     visible: true
 
+            footprints-late:
```

Comment:
JS functions are not allowed in the `visible` property. 

I am rewriting the surrounding footprint/extrude logic to reduce computation at the same time. Will push in a bit.